src/openrct2/world/map.c

```diff
@@ -3168,11 +3168,16 @@ void game_command_place_large_scenery(sint32* eax, sint32* ebx, sint32* ecx, sin
 
 sint32 map_get_station(rct_map_element *mapElement)
 {
-    sint32 station = (mapElement->properties.track.sequence & 0x70) >> 4;
-
     // prevent out of bounds access
-    clamp(0, station, RCT12_MAX_STATIONS_PER_RIDE - 1);
-    return station;
+    sint32 station = (mapElement->properties.track.sequence & SEQUENCE_GET_STATION) >> 4;
```

Comment:
I think it would be far better if this was corrected when parks are imported (and the assert left in place so we can pick up any other related bugs that could cause this).